src/EnsembleKalmanProcess.jl

```diff
@@ -24,15 +23,16 @@ abstract type Process end
 
 
 """
-    EnsembleKalmanProcess{FT<:AbstractFloat, IT<:Int}
+    struct EnsembleKalmanProcess{FT <: AbstractFloat, IT <: Int, P <: Process}
```

Comment:
Our standard convention for docstrings avoids the type (function, struct,...). You can see this in other packages of CliMA.

Reply:
I did that elsewhere.